class/defaults.yml

```
@@ -1,4 +1,21 @@
 parameters:
   openshift4_networking:
     =_metadata: {}
-    namespace: syn-openshift4-networking
+
+    defaultNodeSelector:
+      node-role.kubernetes.io/infra: ''
+
+    patches:
+      ip-reconciler:
+        target:
+          kind: CronJob
+          apiVersion: batch/v1
+          name: ip-reconciler
+          namespace: openshift-multus
```

I wonder if we can remove the repetition here with something like:

```suggestion
      openshift-multus:
        ip-reconciler:
          target:
            kind: CronJob
            apiVersion: batch/v1
```

Not sure how we'd handle cluster-scoped patches then, but I don't think that's something we need to worry about at this point.

---

I have no hard preference here, we can just leave `name` and `namespace` in `target`